rules/override/shellcode.yara

```diff
@@ -0,0 +1,122 @@
+rule shellcode_hash : override {
+    meta:
+        malware_shellcode_hash = "high"
+
```

Comment:
Add a description: for example: are we raising or lowering the severity, and why?

Reply:
Will do. This was mostly just to test that everything was working. I can leave these out of this PR and add them in the next false positive fix PR with the correct details.